VGC/frontend/src/pages/WishlistPage/WishlistPage.css

```diff
@@ -0,0 +1,14 @@
+.page-container {
```

Comment:
Choose better classnames. There could be other pages with similar name so maybe add a wish in front to not mess up the stylings.